models/agent_helpers.go

```diff
@@ -120,12 +134,20 @@ func FindAgentsForNode(q *reform.Querier, nodeID string) ([]*Agent, error) {
 }
 
 // FindAgentsForService returns all Agents providing insights for given Service.
-func FindAgentsForService(q *reform.Querier, serviceID string) ([]*Agent, error) {
+func FindAgentsForService(q *reform.Querier, serviceID string, agentType *AgentType) ([]*Agent, error) {
```

Comment:
`FindAllAgents`, `FindAgentsForService`, `FindAgentsForNode`, `FindAgentsRunningByPMMAgent` - looks almost the same. Do we really need such a zoo of the methods - or it is time to join them in a single one?

Reply:
looks like we can join them